bigquery_etl/cli/monitoring.py

```diff
@@ -288,21 +236,31 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
                 else:
                     bigconfig = BigConfig(type="BIGCONFIG_FILE")
 
-                if (metadata_file.parent / VIEW_FILE).exists():
-                    _update_view_bigconfig(
+                if (
+                    metadata_file.parent / VIEW_FILE
+                ).exists() or "public_bigquery" in metadata.labels:
+                    _update_bigconfig(
```

Comment:
Is the conditional block the same for `else`? Is this intended?

Reply:
todo: leave comment what the difference